src/pages/index.js

```diff
@@ -85,30 +96,47 @@ profilePostButton.addEventListener("click", (evt) => {
   disableButton(modalButtonPost, settings);
```

Comment:
* When the `card modal` opens, there is an error https://snipboard.io/yOrbhG.jpg


* Please, export `disableButton` and `resetValidation` functions from `validation.js`, to use them in `index.js`